exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -153,13 +168,52 @@ def disable_core_dumps():
     resource.setrlimit(resource.RLIMIT_CORE, (0, 0))
 
 
-def sleep_inifinity():
+def sleep_infinity():
     while True:
         time.sleep(1)
 
 
+class User:
+    def __init__(self, user_name: str, group_name: str):
+        self.user_name = user_name
+        self.group_name = group_name
+        self._uid = None
+        self._gid = None
+
+    @property
+    def uid(self):
+        if self._uid is None:
+            self._uid = pwd.getpwnam(self.user_name).pw_uid
+        return self._uid
+
+    @property
+    def gid(self):
+        if self._gid is None:
+            self._gid = grp.getgrnam(self.group_name).gr_gid
+        return self._gid
+
+    def own(self, path: str):
+        if Path(path).exists():
+            unchanged_uid = -1
+            os.chown(path, unchanged_uid, self.gid)
+        return self
+
+    def switch_to(self):
```

Comment:
We also need to add the docker group to the extra groups of the process with

https://docs.python.org/3/library/os.html#os.setgroups

Reply:
I will additionally use `os.setgroups([self.docker_group.id])`